love/src/components/MainTel/M1M3TS/Mixing/Mixing.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ProgressBar from 'components/GeneralPurpose/ProgressBar/ProgressBar';
+import { defaultNumberFormatter } from 'Utils';
+import styles from './Mixing.module.css';
+import message from 'components/ScriptQueue/QueueMessage';
+
+export default class Mixing extends Component {
+  static propTypes = {
+    /** Value of valve position for see in progress bar. */
+    value: PropTypes.number,
+    /** Value of limit position for see in progress bar. (Not is using in this case). */
+    limit: PropTypes.number,
+  };
+  static defaultProps = {
+    value: 0.5,
+    limit: 0.5,
+  };
+
+  render() {
+    const {value, limit} = this.props;
+    return (
+      <>
+      <div className={styles.containerTop}>
+        <span className={styles.title}>Mixing Value</span>
+      </div>
+      <div>
+        <ProgressBar
+          /* targetValue={100} */
```

Comment:
The same as above

Reply:
Will remove it on the next commit. Thanks!